CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,248 @@
+
+## Contract stipulations
+
+### The Basic Grade
+
+(To qualify for this tier, all basic assignment objectives must be completely met.)
+
+* All citizens who meet the covenants of this contract will earn an individual letter grade of `C` in this course
+
+You may ask: why the basic grade is a `C` and not a `B` or `B-`? Rising to the level of above-average or exceptional work relies on the just that -- above-average or exceptional work. Merely completing the assignments maintains a kind of _status quo_. As our interest centers on making a _better_ world, this contract should provide incentive to think beyond mere sufficiency. As you'll see below, work that aspires to higher levels of reward involves doing the work of planning and (better yet) _making_ improvements.
+
+### The `A` tier
+
+(To qualify for this tier, all basic assignment objectives must be completely met.)
+
+Citizens who take the opportunity to both provide _and_ implement substantive improvements to assignment content qualify for this contract tier. In order to qualify for this tier, citizens must:
+
+* propose an assignment-related world improvment in each week's `proposal.md` file which:
+  * completes all document prompts
+  * must be done by start of a lab session (Friday) of a given week
+  * must be approved _in writing_ by the Mayor by the end of the lab session
+  * must either incorporate or provide rationale for _not_ incorporating suggested revisions
+    * this work is due by the assignment deadline (Sunday of each week)
+* implement the improvement proposed by the assignment deadline
+
+These proposals may be revisions to existing content or creation of wholly new content. In many cases, executing planned improvements will require some consultation with either community Technical Leaders _or_ the Mayor. Citizens wishing to achieve this tier should expect to avail themselves of the resources available, including office hours and other forms of correspondence.
+
+Proposals and improvements must be completed for all weeks unless a week's content specifies otherwise in the accompanying assignment `README`. 
+
+#### `A-` grades
+
+Citizens who do not meet the above requirements for two (2) weeks will be awarded a mark of `A-` if all other weeks contain a proposal and subsequent improvement(s).
+
+### The `B` tier
+
+(To qualify for this tier, all basic assignment objectives must be completely met.)
+
+Citizens who take the opportunity to both provide substantive improvements to assignment content qualify for this contract tier. In order to qualify for this tier, citizens must:
+
+* propose an improvement in each week's `proposal.md` file which:
+  * completes all document prompts
+  * must be finished by the start of a lab session (Friday) of a given week
+  * must be recognized _in writing_ by the Mayor by the end of the lab session
+  * must either incorporate or provide rationale for _not_ incorporating suggested revisions
```

Comment:
If we want to propose an improvement, we should provide rationale "to" incorporate suggested revisions, as opposed to "not" incorporating suggested.

Reply:
The intention here is to allow students to choose not to incorporate mayoral suggestions. I'll update the language; I think that clarifies what I mean here.